test/specs/basic.js

```diff
@@ -1,11 +1,31 @@
 const assert = require('assert');
-let should = require('chai').should()
-import loginPage from '../pages/Login.page';
-
-describe('Navigate to archmule page', () => {
-  it('should have the right title', () => {
-    browser.url('https://archmule.com/');
-    const title = browser.getTitle();
-    title.should.be.equal('Archmule');
+let should = require('chai').should();
+
+
+describe('Login and logout of the archmule page', () => {
+  it('should have the right user and password credentials', () => {
+    browser.url('https://archmule.com/login');
+
+    let mailUserTxt = $('#identifier');
```

Comment:
nice practice, but all locators should be located outside all 'describe' sections